packages/create-vite/template-react-ts/README.md

```diff
@@ -25,17 +31,15 @@ export default tseslint.config({
 })
 ```
 
-- Replace `tseslint.configs.recommended` to `tseslint.configs.recommendedTypeChecked` or `tseslint.configs.strictTypeChecked`
-- Optionally add `...tseslint.configs.stylisticTypeChecked`
-- Install [eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react) and update the config:
+You can also install [eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react) for React-specific lint rules:
```

Comment:
Maybe we can already mention it? Even if the code example is still for `eslint-plugin-react`
```suggestion
You can also install [eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react) or [[eslint-plugin-react-x](eslint-plugin-react-x](https://github.com/Rel1cx/eslint-react) for React-specific lint rules:
```

Reply:
Yeah maybe it's not a lot of work left than I expected. I wanted to rewrite this part to `eslint-plugin-react-x` only though and suggest a code example for it.

I'll try to get to that later, but if not I think this PR is ok to be merged and I can create another PR later.

Reply:
As you prefer, thanks for doing this!